tests/lightning_qubit/test_measurements_class.py

```diff
@@ -453,6 +447,16 @@ def test_single_return_value(self, measurement, observable, lightning_sv, tol):
             pytest.skip(
                 f"Measurement of type {type(measurement).__name__} does not have a keyword argument 'wires'."
             )
+        rtol = 1.0e-2  # 1% of expected value as tolerance
+        if shots != None and measurement is qml.expval:
+            # Increase the number of shots
+            if isinstance(shots, int):
+                shots *= 10
+            else:
+                shots = [i * 10 for i in shots]
+
+            # Extra tolerance
+            rtol = 5.0e-2  # 5% of expected value as tolerance
```

Comment:
For this test, the shot number was reduced for `qml.probs` and `qml.var` by one order of magnitude and the absolute tolerance increased by one order of magnitude for the double precision case. However, for `qml.expval`, it was not possible to reduce the shorts and the relative tolerance increased from 0.01 to 0.05. 
Aditionally, the `flaky` `max_runs` pass from 5 to 15 to ensure the success of the test and, at the same time, the total time, running locally, passes from 3mins to 2mins  @multiphaseCFD @AmintorDusko @mlxd

Reply:
Sound good for now. We can revisit this later if sensible. Thank you, @LuisAlfredoNu!